src/app/workflow/descriptors/descriptors.component.html

```diff
@@ -14,9 +14,12 @@
   ~    limitations under the License.
   -->
 <div class="p-3">
-  <alert *ngIf="!_selectedVersion?.valid" type="warning">
-    <mat-icon>warning</mat-icon>
-    &nbsp;This version of the workflow is invalid.
+  <alert *ngIf="validationMessage !== null">
```

Comment:
alert duplicated 3.5 times with this weird one using glyphicon.  Might warrant creating a component just for this.  <alert> is ngx-bootstrap, see if that can be converted to Material if possible.

Reply:
I think the component is out of the scope of this issue, however I do think one would be useful. Maybe one can be created with material.